### ISSUE TITLE
Question: Can this be used via an SPA and without Auth0 Lock?

### ISSUE THREAD

**Issue by michaelpumo** [graphcool/templates#139]
I followed the steps in the README and I can get it working, but I'm just wondering if anyone has an example of this working via an SPA without Lock. 

I am wanting to auth0-js to roll my own custom login screen.

Sorry if this is the wrong place for this. Thanks.

**Comment by maxdarque** [comment]
Yes it's very easy to implement your own auth - see code [here](https://github.com/graphcool/templates/tree/master/auth/email-password)

Graphcool normally ask users to post questions to the [forum](https://www.graph.cool/forum)

**Comment by michaelpumo** [comment]
@maxdarque Sorry, am I missing something? The link you provided appears to integrate to Graph Cool's own email auth and not the Auth0 service. Thanks.

**Comment by maxdarque** [comment]
oh my mistake. I see you want to build your own custom login screen. I've never done that before. I used to use Auth0 then switched to implement my own auth. I know you can customise the colors and logo on Lock that's all

**Comment by michaelpumo** [comment]
@maxdarque Yep. I got it all working, but not with resolvers. I had it going okay when I was using the Auth0 integration but it's now deprecated so I'm trying to move to the resolver solution.
